css/style.css

```diff
@@ -68,6 +68,6 @@ ul {
     line-height: 1.6;
   }
   
-  li {
-    margin: 16px 0;
-  }
+li {
+  margin: 16px 0;
+}
```

Comment:
### Linters errors
you should fix all linters errors. 
you have just solved a part